drivers/include/at30ts74.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2016 University of California, Berkeley
+ * Copyright (C) 2016 Michael Andersen <m.andersen@cs.berkeley.edu>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+/**
+ * @defgroup    drivers_at30ts74 Atmel AT30TS74 temperature sensor
+ * @ingroup     drivers_sensors
+ *
+ * This chip is similar to the AT30TSE75x, but it does not have an EEPROM
+ *
+ * @{
+ *
+ * @file
+ * @brief       Driver for the AT30TS74 ±1°C Accurate, 12-Bit Digital I2C
+ *              Temperature Sensor from Atmel
+ *
+ * @author      Michael Andersen <m.andersen@cs.berkeley.edu>
+ */
+
+#ifndef AT30TS74_H_
+#define AT30TS74_H_
+
+#include <stdint.h>
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name AT30TS74 I2C address
+ * @{
+ */
+#define AT30TS74_ADDR                   (0x49)
+/** @} */
+
+/**
+ * @name AT30TS74 Valid bits options
+ * @{
+ */
+typedef enum {
+  AT30TS74_9BIT  = 0,
+  AT30TS74_10BIT = 1,
+  AT30TS74_11BIT = 2,
+  AT30TS74_12BIT = 3
+} at30ts74_bits_t;
+/** @} */
+
+/**
+  * @brief   Device descriptor for a AT30TS74 device
+  * @{
+  */
+typedef struct {
+  i2c_t   i2c;
```

Comment:
Use a specific struct for params. See other drivers implementations (example: bmp180)